src/main/java/chessmaster/game/Game.java

```diff
@@ -39,6 +57,7 @@ public Game(Color playerColour, ChessBoard board, Storage storage) {
     }
 
     public void run() {
+        TextUI.printText(START_HELP_STRINGS);
         board.showChessBoard();
```

Comment:
Consider creating an instance of `HelpCommand` and executing it instead of duplicating the help messages.
```java
HelpCommand startingHelp = new HelpCommand();
startingHelp.execute(board);
```

Reply:
Will ignore this since HelpCommand prints "looks like you need help" at the start